src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -785,6 +789,75 @@
         }
     }
 
+    .datagrid-compact {
+        .datagrid-column .datagrid-column-separator {
+            &::after {
+                height: calc(100% + 0.5 * #{$clr-datagrid-column-separator-expandby} - #{$clr-default-borderwidth});
+                top: calc(-0.25 * #{$clr-datagrid-column-separator-expandby} + 1px);
+            }
+            .datagrid-column-handle-tracker {
+                top: 0.5*$clr-datagrid-column-handle-tracker-top;
+            }
+        }
+        .datagrid-cell {
+            clr-icon {
+                margin-top: -0.125rem;
+                margin-bottom: -0.125rem;
+            }
+            .badge {
+                margin-top: -0.125rem;
+                margin-bottom: -1px;
+            }
+        }
+        .datagrid-expandable-caret {
+            .spinner {
+                margin-top: 0.125rem;
+            }
+            button {
+                height: 1rem;
+                width: 1rem;
+                outline-offset: $clr-datagrid-compact-outline-offset;
+            }
+            clr-icon {
```

Comment:
Same comment as above regarding the `clr-icon` selector.

Reply:
I think I can target this with a class as this one is inside a fixed datagrid cell.